flume-ng-doc/sphinx/FlumeUserGuide.rst

```diff
@@ -4944,6 +4942,176 @@ Monitoring in Flume is still a work in progress. Changes can happen very often.
++--------------------------+-------------------+----------------+-----------+-----------+---------+--------+
+| AppendBatchAcceptedCount | x                 | x              |           |           | x       | x      |
++--------------------------+-------------------+----------------+-----------+-----------+---------+--------+
+| AppendBatchReceivedCount |                   | x              |           |           | x       | x      |
++--------------------------+-------------------+----------------+-----------+-----------+---------+--------+
+| AppendReceivedCount      |                   |                |           |           |         | x      |
++--------------------------+-------------------+----------------+-----------+-----------+---------+--------+
+| ChannelWriteFail         | x                 | x              | x         | x         | x       | x      |
++--------------------------+-------------------+----------------+-----------+-----------+---------+--------+
+| EventAcceptedCount       | x                 | x              | x         | x         | x       | x      |
++--------------------------+-------------------+----------------+-----------+-----------+---------+--------+
+| EventReadFail            |                   | x              | x         | x         | x       |        |
++--------------------------+-------------------+----------------+-----------+-----------+---------+--------+
+| EventReceivedCount       |                   | x              | x         | x         | x       | x      |
++--------------------------+-------------------+----------------+-----------+-----------+---------+--------+
+| GenericProcessingFail    |                   | x              |           |           | x       |        |
++--------------------------+-------------------+----------------+-----------+-----------+---------+--------+
+| KafkaCommitTimer         |                   |                |           |           |         |        |
++--------------------------+-------------------+----------------+-----------+-----------+---------+--------+
+| KafkaEmptyCount          |                   |                |           |           |         |        |
++--------------------------+-------------------+----------------+-----------+-----------+---------+--------+
+| KafkaEventGetTimer       |                   |                |           |           |         |        |
++--------------------------+-------------------+----------------+-----------+-----------+---------+--------+
+| OpenConnectionCount      |                   |                |           |           |         |        |
++--------------------------+-------------------+----------------+-----------+-----------+---------+--------+
+
+Sinks 1
+~~~~~~~
+
++------------------------+-------------+------------+--------------+---------------+-------+--------+
+|                        | AbstractRpc | AsyncHBase | BucketWriter | ElasticSearch | HBase | HBase2 |
```

Comment:
I guess the BucketWriter is the HDFS sink and the AbstractRpc means some other implementation that will be in the metrics. Could you please check it?

Reply:
Thanks for spotting. Corrected them.